charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-node-windows.yaml

```diff
@@ -1,3 +1,30 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-azuredisk-node-win-config
```

Comment:
We should use the name `{{ .Values.windows.dsName }}-config` to enable side-by-side installs in the future.